src/RTCDTMFSender.ts

```diff
@@ -0,0 +1,27 @@
+import { NativeModules, Platform } from "react-native";
+
+const { WebRTCModule } = NativeModules;
+const DEFAULT_DURATION = 300;
+const DEFAULT_INTER_TONE_GAP = 70;
+
+export default class RTCDTMFSender {
+    _peerConnectionId: number;
+    constructor(peerConnectionId: number) {
+        this._peerConnectionId = peerConnectionId;
+    }
+
+    get canInsertDTMF() {
+        return !!this._peerConnectionId;
+    }
+
+    insertDTMF(tones: string, duration?: number, interToneGap?: number) {
+        const dur = duration || DEFAULT_DURATION;
```

Comment:
Use default values in the function definition instead

Reply:
Done